client/settings/wcpay-settings-context.js

```diff
@@ -11,6 +11,7 @@ const WCPaySettingsContext = createContext( {
 		isAuthAndCaptureEnabled: false,
 		isDisputeIssuerEvidenceEnabled: false,
 		woopay: false,
+		isStripeEceEnabled: false,
```

Comment:
I figured it'd make sense to initialize the context with ECE disabled since that's the default for now. It may make sense to flip this to `true` if/when we start using ECE by default? I don't think it really matters though.

Reply:
Yeah, I don't think it really matters. WooPay is still `false`, for example, so it should be totally fine setting it as `false` by default.